src/osd/ReplicatedBackend.cc

```diff
@@ -1055,7 +1060,7 @@ void ReplicatedBackend::issue_op(
 	    shard,
 	    pinfo);
 
-	if (op->op && op->op->pg_trace)
+	if (pg_trace)
```

Comment:
i don't really like this change, as both op->op and op->op->pg_trace.valid() are O(1), and should complete in a jiffy. introducing another variable does not improve the readability.